src/commands/moderation/unban.ts

```diff
@@ -0,0 +1,42 @@
+import { botCache, unban, sendMessage } from "./../../../deps.ts";
+import { Embed } from "./../../utils/Embed.ts";
+import { sendEmbed } from "./../../utils/helpers.ts";
+
+botCache.commands.set(`unban`, {
+  name: `unban`,
+  guildOnly: true,
+  arguments: [
+    {
+      name: "memberId",
+      type: "string",
```

Comment:
I'd love to see a `snowflake` being added here.  https://github.com/Skillz4Killz/gamer-deno/blob/master/src/arguments/snowflakes.ts

You can take the argument from here and add it

Reply:
a snowflake argument will somewhat validate so that it is more likely to be an id instead of any string